<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FragmentEditorToolbar</class>
 <widget class="QWidget" name="FragmentEditorToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="btPencil">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(21, 21, 21);
    color: white;
    border-width: 0px;
    border-radius: 10px;
    font: bold 12px;
    padding: 4px;
}

QPushButton#evilButton:pressed {
    background-color: rgb(9, 36, 78);
}

QPushButton#evilButton:checked {
    background-color: rgb(86, 145, 236);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/pencil-solid.svg</normalon>
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btEraser">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(21, 21, 21);
    color: white;
    border-width: 0px;
    border-radius: 10px;
    font: bold 12px;
    padding: 4px;
}

QPushButton#evilButton:pressed {
    background-color: rgb(9, 36, 78);
}

QPushButton#evilButton:checked {
    background-color: rgb(86, 145, 236);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/eraser-solid.svg</normalon>
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btLinker">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(21, 21, 21);
    color: white;
    border-width: 0px;
    border-radius: 10px;
    font: bold 12px;
    padding: 4px;
}

QPushButton#evilButton:pressed {
    background-color: rgb(9, 36, 78);
}

QPushButton#evilButton:checked {
    background-color: rgb(86, 145, 236);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/link-solid.svg</normalon>
      </iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>3</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btShowNext">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(21, 21, 21);
    color: white;
    border-width: 0px;
    border-radius: 10px;
    font: bold 12px;
    padding: 4px;
}

QPushButton#evilButton:pressed {
    background-color: rgb(9, 36, 78);
}

QPushButton#evilButton:checked {
    background-color: rgb(86, 145, 236);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/search-solid.svg</normalon>
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btShowLinkedTiles">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(21, 21, 21);
    color: white;
    border-width: 0px;
    border-radius: 10px;
    font: bold 12px;
    padding: 4px;
}

QPushButton#evilButton:pressed {
    background-color: rgb(9, 36, 78);
}

QPushButton#evilButton:checked {
    background-color: rgb(86, 145, 236);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/link-solid.svg</normalon>
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btShowUnlinkedTiles">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(21, 21, 21);
    color: white;
    border-width: 0px;
    border-radius: 10px;
    font: bold 12px;
    padding: 4px;
}

QPushButton#evilButton:pressed {
    background-color: rgb(9, 36, 78);
}

QPushButton#evilButton:checked {
    background-color: rgb(86, 145, 236);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/link-slash-solid.svg</normalon>
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btShowGrid">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(21, 21, 21);
    color: white;
    border-width: 0px;
    border-radius: 10px;
    font: bold 12px;
    padding: 4px;
}

QPushButton#evilButton:pressed {
    background-color: rgb(9, 36, 78);
}

QPushButton#evilButton:checked {
    background-color: rgb(86, 145, 236);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/table-cells-large-solid.svg</normalon>
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
